admin/banner_statistics.php

```diff
@@ -314,7 +314,7 @@ function showTooltip(x, y, contents) {
         </script>
       </div>
 
-<?php if (FALSE) { ?>
+<?php if (FALSE) { //https://github.com/zencart/zencart/issues/5016 ?>
```

Comment:
Let's just get rid of this.  We have source control if someone wants to resurrect it.